inference-engine/src/vpu/common/include/vpu/utils/logger.hpp

```diff
@@ -75,48 +62,41 @@ class Logger final {
     };
 
 public:
-    inline Logger(std::string name, LogLevel lvl, OutputStream::Ptr out) :
+    Logger(std::string name, LogLevel lvl, OutputStream::Ptr out) :
             _name(std::move(name)), _logLevel(lvl), _out(std::move(out)) {
         IE_ASSERT(_out != nullptr);
     }
 
-    inline LogLevel level() const {
```

Comment:
Can we keep it? VPUX plugin rely on this method at this point.

Reply:
I see. Personally, I don't like the idea we need to keep it, but I'll take a look at it, if it can be left as is

Reply:
Dropped a commit with this change from PR. I'm going a ticket to get it back to source tree.